.github/workflows/manual-translation-update.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Manual Translation Update'
+description: 'Manually update translations from Phrase.com and create a PR'
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to update'
+        required: true
+        default: 'master'
+
+jobs:
+  update-translations:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.RELEASE_ACCESS_KEY }}
+
+      - name: Install Phrase CLI
+        uses: phrase/setup-cli@v1.0.2
```

Comment:
[suggestion]: Use SHA commits instead of tags for good security practices.

Reply:
updated thanks!